roles/nova-data/meta/main.yml

```diff
@@ -3,10 +3,11 @@ dependencies:
   - role: inspec
     install_inspec_controls:
       - nova_data
+      - kvm
     tags: inspec
     when:
       - inspec.enabled|bool
-      - inspec.controls.nova_data.enabled|bool
+      - inspec.controls.nova_data.enabled|bool or inspec.controls.kvm.enabled|bool
```

Comment:
oh good catch, I would think the inspec role would handle that since the role itself checks if the item is enabled.